repo/packages/K/kafka/6/config.json

```diff
@@ -51,433 +50,433 @@
         },
         "required":[
           "cpus",
-        "mem",
-        "disk",
-        "count"
-          ]
+          "mem",
+          "disk",
+          "count"
+        ]
       },
       "kafka":{
         "description":"Kafka framework configuration properties",
         "type":"object",
         "additionalProperties":false,
         "properties":{
           "kafka_advertise_host_ip":{
-            "description":"Override advertised.host.name with the IP address detected in /opt/mesosphere/detect_ip",
+            "description":"Automatic configuration of each Broker node's advertised.host.name with the IP address detected in /opt/mesosphere/detect_ip",
             "type":"boolean",
             "default":true
           },
-          "kafka_override_auto_create_topics_enable":{
-            "description":"Override auto.create.topics.enable: Enable auto creation of topic on the server",
+          "auto_create_topics_enable":{
+            "description":"auto.create.topics.enable: Enable auto creation of topic on the server",
```

Comment:
These settings are being passed to Kafka: DCOS/Marathon wants underscores, while Kafka wants periods

https://kafka.apache.org/08/configuration.html